openmc/capi/__init__.py

```diff
@@ -86,6 +86,27 @@ def _error_handler(err, func, args):
     elif err == _error_code('e_tally_invalid_id'):
         raise KeyError("No tally exists with ID={}.".format(args[0]))
 
+    elif err == _error_code('e_invalid_size'):
+        raise MemoryError("Array size mismatch with memory allocated.")
+
+    elif err == _error_code('e_cell_no_material'):
+        raise GeometryError("Operation on cell requires that it be filled"
+                            " with a material.")
+
+    elif err = _error_code('e_below_min_bound'):
+        raise MemoryError("Data has not been loaded beyond lower bound "
+                          "of {}.".format(args[0]))
+
+    elif err = _error_code('e_above_max_bound'):
```

Comment:
You need `==`, not `=` on all these guys. Also, do you intend to keep `E_ABOVE_MAX_BOUND` and `E_BELOW_MIN_BOUND` around or are just the warnings sufficient now?